tests/Module/FormValidatorsTest.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace Tests\Module;
+
+use DateTimeImmutable;
+use Packetery\Core\CoreHelper;
+use Packetery\Module\FormValidators;
+use Packetery\Nette\Forms\Controls\TextInput;
+use PHPUnit\Framework\TestCase;
+
+class FormValidatorsTest extends TestCase {
+	/**
+	 * @return array<array<mixed>>
```

Comment:
To je úplně stejné jako `array`. Rozepsal bych podle reality, v signatuře testu to typované je.